File: TransCoder_tokenized_test_set_functions/python/SUM_MIDDLE_ROW_COLUMN_MATRIX.py
def middlesum ( mat, n ) :
    row_sum = 0
    col_sum = 0
    for i in range ( n ) :
        row_sum += mat [ n //2][i]
    print("Sum of middle row = ",row_sum)
    foriinrange(n):
        col_sum+=mat[i][n//2]
    print("Sum of middle column = ",col_sum)

File: TransCoder_tokenized_test_set_functions/java/SUM_MIDDLE_ROW_COLUMN_MATRIX.java
static void middlesum ( int mat [ ] [ ], int n ) {
  int row_sum = 0, col_sum = 0;
  for ( int i = 0;
  i < n;
  i ++ ) row_sum += mat [ n / 2 ] [ i ];
  System . out . println ( "Sum of middle row = " + row_sum );
  for ( int i = 0;
  i < n;
  i ++ ) col_sum += mat [ i ] [ n / 2 ];
  System . out . println ( "Sum of middle column = " + col_sum );
}